packages/react-components/react-charts-preview/library/src/components/CommonComponents/useCartesianChartStyles.styles.ts

```diff
@@ -200,6 +202,7 @@ export const useCartesianChartStyles_unstable = (props: ICartesianChartProps): I
     axisTitle: mergeClasses(cartesianchartClassNames.axisTitle, baseStyles.axisTitle /*props.styles?.axisTitle*/),
     xAxis: mergeClasses(cartesianchartClassNames.xAxis, baseStyles.xAxis /*props.styles?.xAxis*/),
     yAxis: mergeClasses(cartesianchartClassNames.yAxis, baseStyles.yAxis /*props.styles?.yAxis*/),
```

Comment:
merge yaxisg with yaxis only. Dont create a separate class for it.

Reply:
Okay